src/python/gudhi/simplex_tree.pxd

```diff
@@ -66,6 +66,9 @@ cdef extern from "Simplex_tree_interface.h" namespace "Gudhi":
         vector[Simplex_tree_simplex_handle].const_iterator get_filtration_iterator_end() nogil
         Simplex_tree_skeleton_iterator get_skeleton_iterator_begin(int dimension) nogil
         Simplex_tree_skeleton_iterator get_skeleton_iterator_end(int dimension) nogil
+        # Expansion with blockers
+        ctypedef bool (*blocker_func)(vector[int], void *user_data)
```

Comment:
Please don't use blocker_func both as a type here and as the name of a variable in another file. Maybe add `_t` for the type?

Reply:
Renamed on f4f5992